tool/tsh/aws.go

```diff
@@ -364,6 +348,71 @@ func (a *awsApp) startLocalForwardProxy(port string) error {
 	return nil
 }
 
+// SetupKeyStore installs local CA to the specified Java KeyStore.
```

Comment:
What is the format of a "Java Keystore"? Is it PKCS#12? Proprietary?

Reply:
It should be PKCS12 by default starting Java 9. Though, I couldn't get `openssl pkcs12` on my JDK's default KeyStore.

Reply:
Let's try to name it and call it out the best we can, this kind of info is helpful.

Reply:
A cursory search led me to this: https://docs.oracle.com/javase/8/docs/technotes/guides/security/StandardNames.html#KeyStore (it's pretty old, though).

If it's jks or jceks, let's spell it out.

Reply:
Personally, I don't think we need to know the format. Different versions of Java may use different formats but the `keytool` included in the install should always match/work with the KeyStore (at least the default) the Java developers will be using for their programs.

The `keytool` does support other formats through the `-storetype` flag though.

What do you think we call it `JavaKeyStore` here and I will rename the path flag to`--java-keystore`, but I will also add an extra flag to this tsh command like `--java-keystore-type` and default that to `PKCS12`? That way it's clear what format is going to be used for the keystore and users also get an option to change it if needed.

Reply:
The problem with saying "JavaKeyStore" is that it's not a format, but a bag of varying implementations. It sounds like by "JavaKeyStore" we actually mean "keystores that keytool is able to manage", which is still open-ended but a bit more precise.

My suggestion is to sum up the changes at a high level and take it to product/UX for feedback, so we are not making UX decisions in isolation. There's some nuance here on how to find and handle `keytool`, `keytool --storetype`, whether we should even do this in tsh or just print instructions, etc.

Reply:
Sounds good. I will drop that part then from this PR and review it with the product team. I will update the verify part so we can get that fix first.